apps/backend/api/graphql/mutations/event.js

```diff
@@ -13,30 +18,35 @@ export async function respondToEvent (userId, eventId, response) {
     throw new GraphQLError('Event not found')
   }
 
-  const eventInvitation = await EventInvitation.find({ userId, eventId })
+  let eventInvitation = await EventInvitation.find({ userId, eventId })
+  const sendEmail = (!eventInvitation && going(response)) ||
```

Comment:
seems like this could error out if !eventInvitation and !going(response), because the event invitation would not exist, but I guess that should never happen? would be safest to add question marks like !going(eventInvitation?.get('response') just in case